invision-app/app/components/home/home.js

```diff
@@ -8,17 +8,14 @@ import { MovieList } from '../movieList/movieList';
 import moviesData from '../../model/moviesData.json'
 import { MoivesFound } from "../moviesFound/moviesFound";
 import { MovieListFallback } from "../movieListFallback";
+import {MovieDetails} from '../movieDetails';
 
-export class Home extends React.Component {
+export function Home(props) {
+    const [movies, setMovies] = useState(moviesData);
+    const [movie, setMovie] = useState({});
+    const[isDetailsPage,setDetialsPage]=useState(false);
+    function sort(selectedValue) {
```

Comment:
**sort** does not work because you again mutate state variable. Try to copy **movies** and sort this new list of movies. For example, ```let sortedMovies = [...movies]```

Reply:
Modified.